ray-operator/config/samples/ray-cluster-fluentbit-sidecar.yaml

```diff
@@ -1,26 +1,3 @@
-apiVersion: v1
```

Comment:
Rename the file to `ray-cluster.fluentbit.yaml`. Btw, sidecar container has a new definition since K8s v1.29. It is a special init container https://kubernetes.io/docs/concepts/workloads/pods/sidecar-containers/#pod-sidecar-containers.

Reply:
Do I need to switch to using `initContainers`? I think It might not work for clusters earlier than v1.29.